src/CssInliner.php

```diff
@@ -588,7 +623,21 @@ private function collateCssRules(CssDocument $parsedCss): array
 
             $mediaQuery = $cssRule->getContainingAtRule();
             $declarationsBlock = $cssRule->getDeclarationAsText();
-            foreach ($cssRule->getSelectors() as $selector) {
+            $selectors = $cssRule->getSelectors();
+
+            // Maybe exclude CSS selectors
+            if (!empty($this->excludedCssSelectors)) {
```

Comment:
Do we really need this condition? Or could we rely on the code below to be a no-op for no excluded selectors?

Reply:
I guess we do, otherwise we'll be running `preg_replace` on all selectors for no valid reason.